pkg/parse/event_handler.go

```diff
@@ -123,20 +123,12 @@ func (s *EventHandler) Handle(event events.Event) events.Result {
 		runResult = runFn(s.Context, s.Reconciler, namespaceEvent)
 
 	case events.RetrySyncEventType:
-		// Retry if there was an error, conflict, or any watches need to be updated.
+		// Retry if there was an error, or conflict.
```

Comment:
Is conflict handled by the retry event? From the code, it sounds like conflict is handled by `events.SyncEventType`.

Reply:
Right. I changed this comment before adding the conflict channel. Need to go fix the comment.